### ISSUE TITLE
New release/tag for pre-commit

### ISSUE THREAD

**Issue by egecetin** [cmake-lint/cmake-lint#35]
Recently pre-commit ci begins to fail because of cmake-lint with following error. I saw you already replaced `imp` with #26 but since there is no release/tag the latest release version is failing. Do you plan a new release soon?

```
build attempt 1...
    => error
    [INFO] Installing environment for https://github.com/cmake-lint/cmake-lint.
    [INFO] Once installed this environment will be reused.
    [INFO] This may take a few minutes...
    An unexpected error has occurred: CalledProcessError: command: ('/pc/clone/7U8rx2J5QlOhSHAb03KSLw/py_env-python3/bin/python', '-mpip', 'install', '.')
    return code: 1
    stdout:
        Processing /pc/clone/7U8rx2J5QlOhSHAb03KSLw
          Preparing metadata (setup.py): started
          Preparing metadata (setup.py): finished with status 'error'
    stderr:
          error: subprocess-exited-with-error
      
          × python setup.py egg_info did not run successfully.
          │ exit code: 1
          ╰─> [6 lines of output]
              Traceback (most recent call last):
                File "<string>", line 2, in <module>
                File "<pip-setuptools-caller>", line 34, in <module>
                File "/pc/clone/7U8rx2J5QlOhSHAb03KSLw/setup.py", line 3, in <module>
                  import imp
              ModuleNotFoundError: No module named 'imp'
              [end of output]
      
          note: This error originates from a subprocess, and is likely not a problem with pip.
        error: metadata-generation-failed
    
        × Encountered error while generating package metadata.
        ╰─> See above for output.
    
        note: This is an issue with the package mentioned above, not pip.
        hint: See above for details.
    Check the log at /pc/pre-commit.log
build attempt 2...
    => error
    [INFO] Installing environment for https://github.com/cmake-lint/cmake-lint.
    [INFO] Once installed this environment will be reused.
    [INFO] This may take a few minutes...
    An unexpected error has occurred: CalledProcessError: command: ('/pc/clone/7U8rx2J5QlOhSHAb03KSLw/py_env-python3/bin/python', '-mpip', 'install', '.')
    return code: 1
    stdout:
        Processing /pc/clone/7U8rx2J5QlOhSHAb03KSLw
          Preparing metadata (setup.py): started
          Preparing metadata (setup.py): finished with status 'error'
    stderr:
          error: subprocess-exited-with-error
      
          × python setup.py egg_info did not run successfully.
          │ exit code: 1
          ╰─> [6 lines of output]
              Traceback (most recent call last):
                File "<string>", line 2, in <module>
                File "<pip-setuptools-caller>", line 34, in <module>
                File "/pc/clone/7U8rx2J5QlOhSHAb03KSLw/setup.py", line 3, in <module>
                  import imp
              ModuleNotFoundError: No module named 'imp'
              [end of output]
      
          note: This error originates from a subprocess, and is likely not a problem with pip.
        error: metadata-generation-failed
    
        × Encountered error while generating package metadata.
        ╰─> See above for output.
    
        note: This is an issue with the package mentioned above, not pip.
        hint: See above for details.
    Check the log at /pc/pre-commit.log
```
